lib/beaker/dsl/install_utils/foss_utils.rb

```diff
@@ -845,6 +845,11 @@ def install_puppetlabs_release_repo_on( hosts, repo = nil, opts = options )
             when /^(fedora|el|centos|sles|cisco)$/
               variant_url_value = (($1 == 'centos') ? 'el' : $1)
               variant_url_value = 'cisco-wrlinux' if variant == 'cisco'
+              if variant == 'cisco'
+                version = '5' if version == 'nexus'
+              else
+                version = '7' if version == 'ios_xr'
```

Comment:
I'll remove the if.  The versions are used by beaker to get the correct version of the package.  Our package names are 'cisco-wrlinux-<version>-noarch.rpm'  So we need that to install the package correctly.